superset-frontend/plugins/legacy-preset-chart-deckgl/src/layers/Polygon/Polygon.tsx

```diff
@@ -125,7 +132,11 @@ export function getLayer(
       number,
       number,
     ]) || [0, 0, 0, 0];
-    if (selected.length > 0 && !selected.includes(d[fd.line_column])) {
+    if (
+      selected &&
+      selected.length > 0 &&
+      !selected.includes(d[fd.line_column])
```

Comment:
Can we shorten this up too? 

`!ensureIsArray(selected).includes(d[fd.line_column])`